easybuild/easyconfigs/m/Mako/Mako-1.0.7-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'Mako'
+version = '1.0.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.makotemplates.org'
+description = """A super-fast templating language that borrows the best ideas from the existing templating languages"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = [
```

Comment:
@Helios07 For easyconfigs with just a single source file, we usually collapse this to a single line:

```python
checksums = ['4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae']